docs/build/eps.md

```diff
@@ -358,31 +358,42 @@ See more information on the QNN execution provider [here](../execution-providers
 {: .no_toc }
 * Qualcomm AI Engine Direct SDK (Qualcomm Neural Network SDK) [Linux/Android/Windows](https://qpm.qualcomm.com/main/tools/details/qualcomm_ai_engine_direct)
 
+### Build Options
+{: .no_toc }
+
+* `--use_qnn [QNN_LIBRARY_KIND]`: Builds the QNN Execution provider. `QNN_LIBRARY_KIND` is optional and specifies whether to build the QNN Execution Provider as a shared library (default) or static library.
+  * `--use_qnn` or `--use_qnn shared_lib`: Builds the QNN Execution Provider as a shared library.
+  * `--use_qnn static_lib`: Builds QNN Execution Provider as a static library linked into ONNX Runtime. This is required for Android builds.
+* `--qnn_home QNN_SDK_PATH`: The path to the Qualcomm AI Engine Direct SDK.
+  * Example on Windows: `--qnn_home 'C:\Qualcomm\AIStack\QAIRT\2.31.0.250130'`
+  * Example on Linux: `--qnn_home /opt/qcom/aistack/qairt/2.31.0.250130`
+* `--build_wheel`: Enables Python bindings and builds Python wheel.
+* `--arm64`: Cross-compile for ARM64.
```

Comment:
should we also mention arm64ec?

Reply:
also --buildasx?

Reply:
I added `--arm64ec` to this list, but not `--buildasx` because it's use is a bit complicated (requires building ORT twice via the `build_arm64x.bat` script). However, I also added full build commands for Arm64EC and Arm64X builds below.